paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PaymentOptionsStateMapperTest.kt

```diff
@@ -65,13 +68,15 @@ class PaymentOptionsStateMapperTest {
         )
 
         mapper().test {
+            assertThat(awaitItem()).isEqualTo(PaymentOptionsState())
+
             val cards = PaymentMethodFixtures.createCards(2)
             paymentMethodsFlow.value = cards
             currentSelectionFlow.value = PaymentSelection.Saved(cards.first())
             googlePayStateFlow.value = GooglePayState.Available
             isLinkEnabledFlow.value = true
 
-            assertThat(expectMostRecentItem()?.items).containsNoneOf(
+            assertThat(awaitItem().items).containsNoneOf(
```

Comment:
I replaced `expectMostRecentItem` with `awaitItem` because now we're only emitting when we want with the state flow.